cds/modules/previewer/templates/cds_previewer/video.html

```diff
@@ -0,0 +1,81 @@
+{# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2017 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+#}
+
+{%- extends config.PREVIEWER_ABSTRACT_TEMPLATE %}
+
+{# Configuration Variables #}
+{% set theo_config = [] %}
+{% set video_classes = '' %}
+
+{%- block javascript %}
+  {{ super() }}
+  <script type='text/javascript'
+          src='//cdn.theoplayer.com/latest/c5692bfe-c2a2-4b87-a160-5e1c57e4a16a/theoplayer.loader.js'>
```

Comment:
Can we add it to a config var? IMHO the ``code`` should be secret and set on our infrastructure, WDYT? :)